app/facade.py

```diff
@@ -3,20 +3,22 @@
 from contextlib import asynccontextmanager
 
 import requests
-from aries_cloudcontroller import AriesAgentController, AriesTenantController
+from aries_cloudcontroller import AriesAgentController, AriesTenantController, AriesAgentControllerBase
 from fastapi import Header, HTTPException
 from schemas import LedgerRequest, PostLedgerResponse
 
 admin_url = os.getenv("ACAPY_ADMIN_URL")
 admin_port = os.getenv("ACAPY_ADMIN_PORT")
 is_multitenant = os.getenv("IS_MULTITENANT", False)
 
+admin_api_key = os.getenv("ACAPY_ADMIN_API_KEY")
 
 logger = logging.getLogger(__name__)
 
 
+# TODO: address the way the controller is created
 @asynccontextmanager
-async def create_controller(req_header: Header):
+async def create_controller(place_holder) -> AriesAgentControllerBase:
```

Comment:
need to do some work in this context - defoes need to introduce that fast api dependency